.github/workflows/analyse-pr.yml

```diff
@@ -53,6 +53,18 @@ jobs:
           mvn -f dhis-2/pom.xml clean install -Psonarqube -Pjdk11
           mvn -f dhis-2/pom.xml sonar:sonar -Dsonar.scm.revision=${{ github.event.pull_request.head.sha }} -Dsonar.projectKey=dhis2_dhis2-core -Dsonar.coverage.jacoco.xmlReportPaths=$(find "$(pwd)" -path '*jacoco.xml' | sed 's/.*/&/' | tr '\n' ',')
 
+      - name: Analyse PR dhis-web
+        env:
+          BASE_BRANCH: ${{ github.base_ref }}
+          BRANCH: ${{ github.ref }}
+          PR: ${{ github.event.number }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        if: github.event_name == 'pull_request'
+        run: |
+          mvn -f dhis-2/dhis-web/pom.xml clean install -Psonarqube -Pjdk11
+          mvn -f dhis-2/dhis-web/pom.xml sonar:sonar -Dsonar.scm.revision=${{ github.event.pull_request.head.sha }} -Dsonar.projectKey=dhis2_dhis2-core -Dsonar.coverage.jacoco.xmlReportPaths=$(find "$(pwd)" -path '*jacoco.xml' | sed 's/.*/&/' | tr '\n' ',')
```

Comment:
Maybe we should use another project key on sonar, Dsonar.projectKey=dhis2_dhis2-core  -> dhis2_dhis2-web?

Reply:
Oh yes, I guess this project has to be created manually? @vilkg

Reply:
The project should be created when you do the first analysis. But wouldn't it be better to have everything in one project?